### PR TITLE
Issue #724 Update onboarding to reflect designs

### PR DESCRIPTION
**Pull request by bensinjin**
Most of this work should be pretty straight forward. I made a decision to reuse the "X" (aka close button) we used elsewhere in the app as it was mentioned on the issue but no one responded. In my opinion is adds consistency and generally it's easier to reuse a component and avoid additional translations. If this is a deal breaker we can easily change it.

I've done testing across iOS and Android, it's looking good to me! I haven't had the time to write the animations for swiping between screens but it's definitely something we can get in at some point.

### CODE REVIEW

File: src/components/swipeable_content/swipeable_content_component.tsx [hunk 1]
```diff
@@ -40,7 +40,12 @@ const buildPanResponder = (currentIndex: number, itemCount: number, setState: Se
     })
 );
 
-const onMoveShouldSetPanResponder = (_event: GestureResponderEvent, _gestureState: PanResponderGestureState): boolean => true;
+const onMoveShouldSetPanResponder = (_event: GestureResponderEvent, gestureState: PanResponderGestureState): boolean => (
+    // For 3d touch devices (newer iPhones) we have problems with tappable items not being tappable when nested in the pan responder.
+    // See https://github.com/facebook/react-native/issues/3082 for more details.
+    // This basically says "your tap must be moving horizontally" in order for us to set the responder.
+    gestureState.dx !== 0 ? true : false
```
Comment (rasmus-storjohann-PG) [inline src/components/swipeable_content/swipeable_content_component.tsx]: This code is difficult to understand, so the comment is helpful. However, I don't see the need for the ternary operator here, just `gestureState.dx !== 0` is better.

**Comment by rasmus-storjohann-PG** [comment]
Start Button on the last screen takes you directly to the HOME screen

**Comment by bensinjin** [comment]
Fixed!

File: src/components/swipeable_content/swipeable_content_component.tsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+const getNextIndex = (currentIndex: number, itemCount: number): number => (
+    currentIndex + 1 < itemCount ? currentIndex + 1 : currentIndex
+);
+
+const getPreviousIndex = (currentIndex: number): number => (
+    currentIndex - 1 >= 0 ? currentIndex - 1 : 0
+);
+
+const isRightToLeftSwipe = (horizontalMovementValue: number): boolean => (
+    horizontalMovementValue < 0 && Math.abs(horizontalMovementValue) > swipeSensitivity
```
Comment (rasmus-storjohann-PG) [inline src/components/swipeable_content/swipeable_content_component.tsx]:
```
horizontalMovementValue < -1.0 * swipeSensitivity
```

File: src/components/swipeable_content/swipeable_content_component.tsx [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+);
+
+const isLeftToRightSwipe = (horizontalMovementValue: number): boolean => (
+    horizontalMovementValue > 0 && Math.abs(horizontalMovementValue) > swipeSensitivity
```
Comment (rasmus-storjohann-PG) [inline src/components/swipeable_content/swipeable_content_component.tsx]:
```
horizontalMovementValue > swipeSensitivity
```

File: src/components/swipeable_content/swipeable_content_component.tsx [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+);
+
+const updateIndexIfChanged = (currentIndex: number, newIndex: number, setState: SetState): void => {
+    if (currentIndex !== newIndex) {
+        setState({ currentIndex: newIndex });
+    }
+};
```
Comment (rasmus-storjohann-PG) [inline src/components/swipeable_content/swipeable_content_component.tsx]: Pretty sure react is smart enough to ignore a setState call that doesn't change the state? Then this function is not needed.

File: src/components/swipeable_content/swipeable_content_component.tsx [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+    const dotSize = 8;
+    const dotStyle = { width: dotSize, height: dotSize, borderRadius: dotSize / 2, marginHorizontal: 5 };
+    if (props.currentIndex === props.loopIndex) {
+        return <View style={{ ...dotStyle, backgroundColor: colors.topaz }} />;
+    }
+    return <View style={{ ...dotStyle, backgroundColor: colors.lightGrey }} />;
+};
+
+const getNextIndex = (currentIndex: number, itemCount: number): number => (
+    currentIndex + 1 < itemCount ? currentIndex + 1 : currentIndex
```
Comment (rasmus-storjohann-PG) [inline src/components/swipeable_content/swipeable_content_component.tsx]:
```
R.min(currentIndex + 1, itemCount - 1)
```

File: src/components/swipeable_content/swipeable_content_component.tsx [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+);
+
+const getPreviousIndex = (currentIndex: number): number => (
+    currentIndex - 1 >= 0 ? currentIndex - 1 : 0
```
Comment (rasmus-storjohann-PG) [inline src/components/swipeable_content/swipeable_content_component.tsx]:
```
R.max(currentIndex - 1, 0)
```

File: src/components/swipeable_content/swipeable_content_component.tsx [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+
+interface SwipeableContentComponentProps {
+    readonly contentItems: ReadonlyArray<JSX.Element>;
+}
+
+interface State {
+    readonly currentIndex: number;
+}
+
+type SetState = React.Dispatch<React.SetStateAction<State>>;
```
Comment (rasmus-storjohann-PG) [inline src/components/swipeable_content/swipeable_content_component.tsx]: You can probably use `Readonly<ReturnType<typeof buildState>>;` here

File: src/components/swipeable_content/swipeable_content_component.tsx (hunk 1 above)
Comment (bensinjin) [reply src/components/swipeable_content/swipeable_content_component.tsx]: Fixed

File: src/components/swipeable_content/swipeable_content_component.tsx (hunk 2 above)
Comment (bensinjin) [reply src/components/swipeable_content/swipeable_content_component.tsx]: Nice and tidy, fixed.

File: src/components/swipeable_content/swipeable_content_component.tsx (hunk 3 above)
Comment (bensinjin) [reply src/components/swipeable_content/swipeable_content_component.tsx]: fixed

File: src/components/swipeable_content/swipeable_content_component.tsx (hunk 4 above)
Comment (bensinjin) [reply src/components/swipeable_content/swipeable_content_component.tsx]: yeah you're right, fixed

File: src/components/swipeable_content/swipeable_content_component.tsx (hunk 5 above)
Comment (bensinjin) [reply src/components/swipeable_content/swipeable_content_component.tsx]: fixed

File: src/components/swipeable_content/swipeable_content_component.tsx (hunk 6 above)
Comment (bensinjin) [reply src/components/swipeable_content/swipeable_content_component.tsx]: fixed

File: src/components/swipeable_content/swipeable_content_component.tsx (hunk 7 above)
Comment (bensinjin) [reply src/components/swipeable_content/swipeable_content_component.tsx]: Dunno about this - TS didn't like "the circular reference" - I don't have time at this minute to dig much further.

**Pull request merged**